src/main/scala/com/gu/commercial/branding/BrandingFinder.scala

```diff
@@ -63,17 +63,21 @@ object SponsorshipHelper {
     dateLaterThanThreshold getOrElse true
   }
 
-  def findSectionSponsorship(edition: String, publishedDate: Option[CapiDateTime])
+  def findSponsorshipFromSection(edition: String, publishedDate: Option[CapiDateTime])
     (section: Section): Option[Sponsorship] = {
     section.activeSponsorships.flatMap(_.find { s =>
       isTargetingEdition(edition)(s) && isTargetingDate(publishedDate)(s)
     })
   }
 
-  def findTagsSponsorship(tags: Seq[Tag], publishedDate: Option[CapiDateTime], edition: String): Option[Sponsorship] =
-    tags.toStream.flatMap(t => findTagSponsorship(edition, publishedDate)(t)).headOption
+  def findSponsorshipFromTags(
+    tags: Seq[Tag],
+    publishedDate: Option[CapiDateTime],
+    edition: String
+  ): Option[Sponsorship] =
+    tags.toStream.flatMap(t => findSponsorshipFromTag(edition, publishedDate)(t)).headOption
```

Comment:
There appears to be no difference in this case.  It's just that I've never used `view` before.   Presumably `view` has to do less work so I'll use that.